index.html

```diff
@@ -0,0 +1,64 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+        href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;700&family=Poppins:wght@300;700&&family=Roboto:wght@700&display=swap"
+        rel="stylesheet" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <header class="navbar">
+        <div class="navbar-wrapper">
+            <a class="my-logo" href="#">My Logo</a>
+            <nav>
+                <img class="hamburger-menu" src="images/hamburger.svg" alt="Hamburger Menu" />
+            </nav>
+        </div>
+    </header>
+    <section class="headline">
+        <div class="headline-wrapper">
+            <h1 class="greeting-title">Hey There.<br />I’m Peter.</h1>
+            <p class="greeting-description1">I'm a Software Developer</p>
+            <p class="greeting-description2">
+                I can help you build a product , feature or website Look through some
+                of my work and experience! If you like what you see and have a project
+                you need coded, don’t hesitate to contact me.
+            </p>
+            <ul class="headline-social-media-links">
+                <li>
+                    <a class="social-media-link" href="#"><img class="headline-link-img" src="images/github.svg"
+                            alt="GitHub Link" /></a>
+                </li>
+                <li>
+                    <a class="social-media-link" href="#"><img class="headline-link-img" src="images/linkedin.svg"
+                            alt="Linkedin Link" /></a>
+                </li>
+                <li>
+                    <a class="social-media-link" href="#"><img class="headline-link-img" src="images/angelist.svg"
+                            alt="Angelist Link" /></a>
+                </li>
+                <li>
+                    <a class="social-media-link" href="#"><img class="headline-link-img" src="images/twitter.svg"
+                            alt="Twitter Link" /></a>
+                </li>
+                <li>
+                    <a class="social-media-link" href="#"><img class="headline-link-img" src="images/medium.svg"
+                            alt="Medium Link" /></a>
+                </li>
+            </ul>
+        </div>
+
+        <img class="illustration" src="images/illustration.svg" alt="illustration" />
```

Comment:
- [ ] kindly as requested by the previous reviewer consider importing this background image from CSS background-image property for best practices. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. and after that make sure that sure your background image does not overlap the body and overflow-x is not possible. That would increase the user experience.